.github/workflows/chore_enforce_changelog_version_bump.yaml

```diff
@@ -0,0 +1,76 @@
+name: "[chore] Enforce Changelog and Version Bump"
+
+on: 
+  pull_request:
+    paths:
+      - 'unique_sdk/**'
+      - 'unique_toolkit/**'
+
+jobs:
+  validate-changes:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # Fetch full history for proper diff comparison
+
+    - name: Determine which package was changed
```

Comment:
Seems like a lot of code execution just to determine which package was changed. I would recommend we use two separate workflows for each and then you don't have to do any/most of the code execution to determine which files were changed.